scripts/dev/nightlybuilds.sh

```diff
@@ -52,7 +52,8 @@ fi
 ./create_package.sh linux64 $lastversion master gcc4
 ./create_package.sh linux64 $lastversion master gcc5
 ./create_package.sh linux64 $lastversion master gcc6
```

Comment:
@arturoc do you think here we should try and programmatically pull the default branch? So during patch-release phases it is targeting patch-release instead of master?  

```
git remote show origin | grep "HEAD branch" | cut -d ":" -f 2
``` 

This outputs `patch-release` currently.

Reply:
yes that would be great but not sure there's a way to do that without using the github api or something like that. i usually just change it manually while we are on a patch release cycle